app/controllers/katello/concerns/api/v2/http_proxies_controller_extensions.rb

```diff
@@ -0,0 +1,17 @@
+module Katello
+  module Concerns
+    module Api
+      module V2
+        module HttpProxiesControllerExtensions
+          extend ::Apipie::DSL::Concern
+
+          update_api(:create) do
+            param :http_proxy, Hash do
+              param :default_content_proxy, :bool, :required => false, :desc => N_('Set this proxy as the default content proxy')
```

Comment:
I think it would be better to avoid confusion between http proxy and smart proxy in APIdoc 
```suggestion
              param :default_content_proxy, :bool, :required => false, :desc => N_('Set this HTTP proxy as the default content HTTP proxy')
```